packages/compass-sidebar/src/components/csfle-connection-modal/csfle-connection-modal.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  css, spacing, Banner, Description, Label, Link, Toggle, Modal
+} from '@mongodb-js/compass-components';
+
+const toggleStyles = css({
+  marginBottom: spacing[3],
```

Comment:
How do you feel about a bit more spacing between the top title and this toggle?
```suggestion
  marginTop: spacing[3],
  marginBottom: spacing[3],
```

Reply:
Done 👍